### ISSUE TITLE
Searching using tags

### ISSUE THREAD

**Issue by gerases** [openannotation/annotator#589]
Hi,

I've set annotator up and it's working great on my locally stored html pages, but when I try to search by the tag using the tag fliter in the toolbar, it doesn't find anything. Searching by annotations works though. What could be the reason?

The JSON does contain the tags and I do see them when I try to edit an entry.

I've initialized the plugin like so:

```
 $('body').annotator();
 $('body').annotator('setupPlugins');
```

Thanks,
  Sergei

**Comment by tilgovi** [comment]
I can't reproduce this. Can you share the page where this is happening or share some screenshots?

**Comment by gerases** [comment]
Hi Randall,

Thanks for getting back to me. I have the annotator installed locally on my
Mac. Everything else works fine. A screenshot is attached. I could try
debugging that particular functionality if you give me some tips on what to
look for.

[image: Inline image 2]

Thanks so much,
   Sergei

**Comment by tilgovi** [comment]
I don't think that came through via email. Could you try it in the Github interface?

**Comment by gerases** [comment]
Sure thing. Here it goes:

![image](https://cloud.githubusercontent.com/assets/8953623/12771297/ecd5059e-c9ed-11e5-8a0a-5acae021df8d.png)

**Comment by gerases** [comment]
After tagging something with `error`, I type `error` into the **Tags** filter and the highlighted word goes away.

**Comment by kowdykung** [comment]
@gerases hi, can I ask you the installation? how to get the tag function? I can only get the highlight to comment to text.
